data/pug.txt

```diff
@@ -192,6 +195,7 @@ outfit "Pug Gridfire Turret"
 		"hull damage" 600
 		"blast radius" 40
 		"piercing" .3
+	description "No one quite knows how the Gridfire Turret works, and the Pug aren't interested explaining it. But it is clearly capable of releasing tremendous energies at any point in space to dice even the strongest of warships."
```

Comment:
"the Pug aren't interested explaining it" - forgot the 'in' after interested.

'Dice' is also a rather awkward word. Esp. since we don;t really know what the Gridfire actually does to its target (for example, I picture it as tearing ships via spacetime manipulation). Perhaps a more neutral word like 'obliterate'?

Reply:
I changed it to disable, better?